src/main/java/teammates/ui/template/InstructorResultsParticipantPanel.java

```diff
@@ -0,0 +1,81 @@
+package teammates.ui.template;
+
+
+public class InstructorResultsParticipantPanel implements Comparable<InstructorResultsParticipantPanel>{
+    private boolean isGiver; // if false, then participant is a recipient
+    private String participantIdentifier; // email, team name, or %GENERAL%
+    private String name; 
+    
+    private boolean isHasResponses;
+    
+    String profilePictureLink;
+    
+    boolean isModerationButtonDisplayed;
```

Comment:
This appears unread in the Java files.
Would a `${empty moderationButton}` with a null value for that suffice to replace this variable in the tags?